### DISCUSSION TITLE
Uneven hard drive size

### DISCUSSION THREAD

**Discussion by macosupdate** [seaweedfs/seaweedfs#3494] [General]
Thank you for creating seaweedfs, it helped me with my storage problem. Wish I knew it sooner.

I'm using seaweedfs for an image hosting service. Because there is quite a lot of traffic. So filer got error "too many open files", I updated to new version, and increased limit ulimit -n. Service is back up.
But it only writes to a single hard drive (data4). I run cmd **volume.balance** in weed shell but nothing happened.
Here is my setup:

- weed master -volumeSizeLimitMB=1024
- weed volume -index=leveldb -max="0,0,0,0,0,0,0" -mserver="localhost:9333" -dir="/data1,/data2,/data3,/data4,/data5,/data6,/data7"
- weed filer
- weed mount -filer=localhost:8888 -dir=/img -filer.path=/

![Screen Shot 2022-08-23 at 07 54 07](https://user-images.githubusercontent.com/65846907/186045742-e869a729-8781-4b2d-b8cb-523c1fc33428.png)

**Comment by chrislusf** [comment]
You can preallocate a few volumes for writing.
